src/Plugin/jsonapi/FieldEnhancer/CardLinkEnhancer.php

```diff
@@ -176,32 +180,33 @@ public function getCardFields($nid) {
   /**
    * Helper function to add image field details.
    *
-   * @param string $nid
-   *   The node id.
+   * @param \Drupal\Core\Entity\EntityInterface $node
+   *   The node entity.
    *
    * @return array
    *   The image data with focal point values.
    */
-  public function getImage($nid) {
+  public static function getImage(EntityInterface $node) {
```

Comment:
Why static function declaration? Do you intend to use CardLinkEnhancer::getImage somewhere?

Reply:
good question? I was supposed to declare this `getCropEntity` function in tide_media as static instead of `getImage ` function here.
pushing a PR now to fix it.